src/main/java/com/example/nasastealphoto/model/Picture.java

```diff
@@ -0,0 +1,38 @@
+package com.example.nasastealphoto.model;
+
+import com.fasterxml.jackson.annotation.JsonManagedReference;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.CreationTimestamp;
+import java.time.LocalDateTime;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "pictures")
+public class Picture {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(unique = true)
+    private Long nasaId;
+
+    @Column(nullable = false)
+    private  String imgSrc;
+
+    @CreationTimestamp
+    private LocalDateTime createdAt;
```

Comment:
The same remark as for "create-tables.sql". We have to parse value for this field from nasa json